tools/internal_ci/linux/grpc_unittests_python_in_docker.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
This seems to be a (near) copy of [this file](https://github.com/grpc/grpc/blob/master/tools/distrib/install_all_python_modules.sh). The `install_all_python_modules.sh` file name and path clearly indicate that it is meant for consumption by developers.

This file, however, lives in `internal_ci` and it has `in_docker` in the name. Shoudl we instead just update `install_all_python_modules.sh` to pull in any needed fixes and then call _it_ from `grpc_unittests.sh`?

Reply:
Updated as suggested.